config/config.py

```
@@ -26,6 +26,10 @@ class Config(BaseSettings):
     DEFAULT_MODEL: str = Field(default="gemini/gemini-1.5-flash", alias="DEFAULT_MODEL")
     API_BASE_URL: Optional[str] = Field(default=None, alias="API_BASE_URL")
 
+    # Ollama Cloud 設定（OpenAI互換API）
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)